ui-admin/src/App.tsx

```diff
@@ -48,7 +48,7 @@ function App() {
       <ConfigConsumer>
         { config =>
           <AuthProvider {...getOidcConfig(config.b2cTenantName, config.b2cClientId, config.b2cPolicyName)}>
-            <I18nProvider>
+            <I18nProvider defaultLanguage={'en'}>
```

Comment:
I think there's some more work to do here, but I wasn't ready to include it in this PR. I'd like to see if we can have a common PortalProvider interface that can be used across the ui-admin and ui-participant, kinda like how we have the common ApiProvider. Without this, the I18nProvider needs to be handled slightly differently depending on the context that it's in